src/javascript/Dealer/Public/DealerRegistration/RegistrationForm.tsx

```diff
@@ -0,0 +1,224 @@
+            className={styles.formInput}
+            // value={params.name}
+            // onChange={handleOnChange}
+            required
+          />
+        </InputWrapper>
+        <InputWrapper className={cn(styles.formItem, 'col-span-6')}>
+          <Label className="sr-only" htmlFor="streetAddress">
+            Organization Address
+          </Label>
+          <Input
+            placeholder="Street Address"
+            id="streetAddress"
+            name="streetAddress"
+            className={styles.formInput}
+            // value={params.streetAddress}
+            // onChange={handleOnChange}
+            required
+          />
+        </InputWrapper>
+        <InputWrapper className={cn(styles.formItem, 'col-span-6')}>
+          <Label className="sr-only" htmlFor="streetAddressTwo">
+            Organization Street Address 2
+          </Label>
+          <Input
+            placeholder="Street Address Line 2"
+            id="streetAddressTwo"
+            name="streetAddressTwo"
+            className={styles.formInput}
+            // value={params.streetAddressTwo}
+            // onChange={handleOnChange}
+          />
+        </InputWrapper>
+        <InputWrapper className={cn(styles.formItem, 'col-span-2')}>
+          <Label className="sr-only" htmlFor="city">
+            City
+          </Label>
+          <Input
+            placeholder="City"
+            id="city"
+            name="city"
+            className={styles.formInput}
+            // value={params.city}
+            // onChange={handleOnChange}
+            required
+          />
+        </InputWrapper>
+        <InputWrapper className={cn(styles.formItem, 'col-span-2')}>
+          <Label className="sr-only" htmlFor="state">
+            State
+          </Label>
+          <Input
+            placeholder="State"
+            id="state"
+            name="state"
+            className={styles.formInput}
+            // value={params.state}
+            // onChange={handleOnChange}
+            required
+          />
+        </InputWrapper>
+        <InputWrapper className={cn(styles.formItem, 'col-span-2')}>
+          <Label className="sr-only" htmlFor="zip">
+            Zip
+          </Label>
+          <Input
+            placeholder="Zip"
+            id="zip"
+            name="zip"
+            maxLength={5}
+            className={styles.formInput}
+            // value={params.zip}
+            // onChange={handleOnChange}
+            required
+          />
+        </InputWrapper>
+        <InputWrapper className={cn(styles.formItem, 'col-span-6')}>
+          <Label className="sr-only" htmlFor="default_dealer_number">
+            Default Dealer Number
+          </Label>
+          <Input
+            placeholder="Default Dealer Number"
+            id="default_dealer_number"
+            name="default_dealer_number"
+            className={styles.formInput}
+            // value={params.default_dealer_number}
```

Comment:
There is a ton of commented out code. What is the rational?

Reply:
Forgot to remove. Copied the code over from the pre-existing components. Removed dead code and removed old component files.